src/Modules/OrchardCore.Commerce/Recipes/OrchardCore.Commerce.Development.Tests.Setup.recipe.json

```diff
@@ -13,6 +13,24 @@
     "test"
   ],
   "steps": [
+    {
+      "name": "settings",
```

Comment:
Please remove the `settings` block from the end, and merge it here. It's just `"UseCdn": false`.